README.md

```diff
@@ -57,6 +57,12 @@ You can add more commands by adding to, or overriding, the variable.
 The nagios services are currently in the "checks" per hostgroup variables, see defaults/main.yml.
 You can add more services by adding to, or overriding, the variable.
 
+### Nagios Custom Checks:
```

Comment:
The readme doesn't say how to use `nagios_custom_checks` I have to read the code to see it's used as a list to tell the template module to template out some files(that come from where?).